### ISSUE TITLE
please use -f option

### ISSUE THREAD

**Issue by opyate** [tutumcloud/tutum-cli#55]
Suggested ```-f``` option doesn't seem to work, and no clear instruction of where it should go:

```
~ tutum image push my_app
Pushing my_app to Tutum private registry ...
Tagging my_app as tutum.co/opyate/my_app ...
409 Client Error: Conflict ("Conflict: Tag latest is already set to image 1f6c9e945776a8631087ef949c12bfd3c6d3dfe9b68227e3b65b3a70ad39c7a4, if you want to replace it, please use -f option")
~  tutum -f image push my_app
usage: tutum [-h] [-v]
             
             {build,container,image,login,node,nodecluster,service,tag,webhook-handler}
             ...
tutum: error: unrecognized arguments: -f
~  tutum image -f push my_app
usage: tutum [-h] [-v]
             
             {build,container,image,login,node,nodecluster,service,tag,webhook-handler}
             ...
tutum: error: unrecognized arguments: -f
~  tutum image push -f my_app
usage: tutum [-h] [-v]
             
             {build,container,image,login,node,nodecluster,service,tag,webhook-handler}
             ...
tutum: error: unrecognized arguments: -f
~  tutum image push my_app -f
usage: tutum [-h] [-v]
             
             {build,container,image,login,node,nodecluster,service,tag,webhook-handler}
             ...
tutum: error: unrecognized arguments: -f
~  tutum -v
tutum 0.11.0.1

```

**Comment by fermayo** [comment]
This has been already fixed in f61bd2ee55d63dfb40b38b0e5333969f9c120fe1 but not yet released. We will publish a new version in the next few hours.

**Comment by opyate** [comment]
Ah, thanks. Should've checked the last few commits before issuing :+1:
